### PR TITLE
Feature/graphs for tag plugins

### LINKED ISSUE #69: Add graphs for plugin TAG
1. PIE graph of used tags
2. PIE graph of used tags in Ticket

### PR DESCRIPTION
**Pull request by flegastelois**
Close #69

### CODE REVIEW

File: inc/tag.class.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * @version $Id: HEADER 15930 2011-10-30 15:47:55Z tsmr $
+ -------------------------------------------------------------------------
+ Mreporting plugin for GLPI
+ Copyright (C) 2003-2011 by the mreporting Development Team.
+
+ https://forge.indepnet.net/projects/mreporting
```
Comment (trasher) [inline inc/tag.class.php]: Wrong URL :)

File: inc/tag.class.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with mreporting. If not, see <http://www.gnu.org/licenses/>.
+ --------------------------------------------------------------------------
+ */
+ 
+class PluginMreportingTag Extends PluginMreportingBaseclass {
+
+   function reportPieTag($config = array()) {
```
Comment (trasher) [inline inc/tag.class.php]: Could you please add phpdoc on new added methods?

File: inc/tag.class.php (hunk 1 above)
Comment (flegastelois) [inline inc/tag.class.php]: All plugin files were affected.
Maybe it's better to do that with another "fix" ?

File: inc/tag.class.php [hunk 3]
```diff
@@ -29,9 +29,18 @@
...
+    * Default pie graph for the use of tags.
+    * For all linked itemtypes without filter.
+    *
+    * @param array   $config (optionnal)
+    * @return array  $datas array of query results (tag => count number)
+    */
    function reportPieTag($config = array()) {
       global $DB;
 
+      ddd();
```
Comment (trasher) [inline inc/tag.class.php]: Huh?

File: inc/tag.class.php (hunk 1 above)
Comment (trasher) [inline inc/tag.class.php]: OK ;)

File: inc/tag.class.php (hunk 3 above)
Comment (flegastelois) [inline inc/tag.class.php]: Oh, my bad ^^

**Pull request merged**